Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -46,6 +46,12 @@ $sourcePath = $sourcePath.Trim('"')
 $storageAccount = $storageAccount.Trim()
 $containerName = $containerName.Trim().ToLower()
 
+# Normalize admin username
+if($vmsAdminUserName -and (-not $vmsAdminUserName.StartsWith(".\")) -and ($vmsAdminUserName.IndexOf("\") -eq -1) -and ($vmsAdminUserName.IndexOf("@") -eq -1))
```

Comment:
Do we need (-not $vmsAdminUserName.StartsWith(".\")) check? I guess this will be coved as part of 2nd clause. In that case, I will remove that check from all the three task

Reply:
Actually this check is not required.  Though we have followed same pattern we are using in remote deployer.